src/backend/common/models/sitevar.py

```diff
@@ -1,6 +1,6 @@
 import json
 
-from google.appengine.ext import ndb
+from google.cloud import ndb
 
 
 class Sitevar(ndb.Model):
```

Comment:
I wonder if we can also subclass from ABC here to force new SVs to define a schema 

https://docs.python.org/3/library/abc.html

Reply:
Sitevar models (like `WebsiteBlacklist`) don't actually subclass `Sitevar`, they just wrap `Sitevar`.